web/yo/app/scripts/services/mainutils.js

```diff
@@ -454,7 +454,36 @@ angular.module('oncokbApp')
                 return false;
             }
         };
-
+        function isProcessed(type, uuid) {
```

Comment:
Either give a more informative name to this function or move the whole thing to`reviewUtils`. Otherwise, it is confusing when using `MainUtils. isProcessed()`